lib/entities/integrations/index.ts

```diff
@@ -0,0 +1,29 @@
+/** @format */
+
+import {buildApiUrl, NetworkClient} from "../../network";
+import {IIntegration} from "./models";
+import {wrapWithErrorHandler} from "../../network/utils";
+
+export class Integrations {
+    networkClient: NetworkClient;
+    baseUrl: string = "integrations";
+
+    constructor(networkClient: NetworkClient) {
+        this.networkClient = networkClient;
+    }
+
+    /**
+     * Retrieve all integrations that matches the specified plan identifier
+     * @param planIdentifier - Identifier of the plan that the integration belongs to.
+     * @returns All integrations on the plan including its relevant information about the app
```

Comment:
The comment for the `@returns` could be adjusted more. Instead of saying "its relevant information about the **app**", we could say "its relevant information about the **integration**" as it relates to the integration, not an application